specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/preview/2022-09-01-preview/vaults.json

```diff
@@ -0,0 +1,2128 @@
+          }
+        }
+      }
+    },
+    "UserIdentity": {
+      "type": "object",
+      "description": "A resource identity that is managed by the user of the service.",
+      "properties": {
+        "principalId": {
+          "description": "The principal ID of the user-assigned identity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "clientId": {
+          "description": "The client ID of the user-assigned identity.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnectionVaultProperties": {
+      "description": "Information to be stored in Vault properties as an element of privateEndpointConnections List.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Format of id subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.[Service]/{resource}/{resourceName}/privateEndpointConnections/{connectionName}.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnection"
+        },
+        "name": {
+          "description": "The name of the private Endpoint Connection",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type, which will be of the format, Microsoft.RecoveryServices/vaults/privateEndpointConnections",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "The location of the private Endpoint connection",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "description": "Private Endpoint Connection Response Properties.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Gets or sets provisioning state of the private endpoint connection.",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Pending"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "privateEndpoint": {
+          "$ref": "#/definitions/PrivateEndpoint"
+        },
+        "privateLinkServiceConnectionState": {
+          "$ref": "#/definitions/PrivateLinkServiceConnectionState"
+        }
+      }
+    },
+    "PrivateEndpoint": {
+      "description": "The Private Endpoint network resource that is linked to the Private Endpoint connection.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Gets or sets id.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "groupIds": {
```

Comment:
>groupIds [](http://example.com/codeflow?start=9&length=8)

This name doesn't seem to match what this array contains.

Reply:
The name of this has been kept so as to be consistent with PrivateEndpoint contract followed Azure-wide. Improved the description to match the values